build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
     id 'java'
     id 'org.jetbrains.intellij' version '0.4.18'
     id 'io.franzbecker.gradle-lombok' version '3.3.0'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.0'
```

Comment:
This is needed because the tests fail because of version mismatch with the kotlin version from the 2020.3 sdk